src/components/FilmList/style.scss

```diff
@@ -0,0 +1,17 @@
+.film-list {
+  width: 90%;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 395px);
+  grid-gap: 40px 20px;
+  justify-content: center;
+}
+
+.film-list__none {
+  height: 500px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-weight: 500;
+  font-size: 50px;
+  color: #f65261;
```

Comment:
there are the same colors across all app. Consider to make some css variables for project.